clients/ts/signalr/src/HttpClient.ts

```diff
@@ -106,6 +115,7 @@ export class DefaultHttpClient extends HttpClient {
                 reject(new TimeoutError());
             };
 
+            this.logger.log(LogLevel.Trace, `xhr.send(${request.content || ""})`);
```

Comment:
Isn't there a flag now for whether request content is displayed in logs in the TS client. Should reuse that logic from the transports. e.g. actual sent content vs content length.

Reply:
This flag https://github.com/aspnet/SignalR/pull/1792?

Reply:
Yup

Reply:
Ok I'm removing logging from this change. I don't want to block waiting for anything.